.idea/misc.xml

```diff
@@ -12,5 +12,8 @@
     </writeAnnotations>
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="liberica-11" project-jdk-type="JavaSDK" />
+  <component name="ProjectPlainTextFileTypeManager">
+    <file url="file://$PROJECT_DIR$/README.markdown" />
```

Comment:
PlainTextFileType? Does this disable inspections and rendering for the README?

Reply:
Yes, it seems to, if I restart after pulling this and open README, I no longer have proper Markdown support. :disappointed:

Reply:
yep.
I not noticed change...
I changed readme... IDEA throws error to me (Needs javafx's webview for rendering) and didn't want open file.
will fixed